src/OSPSuite.R/Services/DataImporter.cs

```diff
@@ -95,7 +95,6 @@ public IList<MetaDataCategory> DefaultMetaDataCategories()
          categories.Add(createMetaDataCategory<string>(Constants.ObservedData.GENDER, isListOfValuesFixed: true));
          categories.Add(createMetaDataCategory<string>(Constants.ObservedData.DOSE));
          categories.Add(createMetaDataCategory<string>(Constants.ObservedData.ROUTE));
-         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.GROUP_ID));
```

Comment:
Why do we duplicate the knowledge of default categories? Shouldn't we abstract that ? Anyways merging but surprising